lib/web/mage/sticky.js

```diff
@@ -11,7 +11,26 @@ define([
 
     $.widget('mage.sticky', {
         options: {
-            container: ''
+            container: '',
+            spacingTop: 0,
```

Comment:
You have been provide new settings properties that by default is Number, no reason to check it on another types. User can't change this property, only developer, when developer will be configuring sticky widget he can look at type of parameter. Also, `offsetTop` it`s native property of HTMLElement would be nice to change it to another, e.g. `blockOffsetTop`.